src/Plugin/Purge/Purger/AkamaiPurger.php

```diff
@@ -93,4 +85,11 @@ public function invalidate(array $invalidations) {
     $this->client->purgeUrls($urls_to_clear);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function hasRuntimeMeasurement() {
+    return TRUE;
+  }
+
```

Comment:
I don't really understand this API change, I guess we need to feed back runtime stats to Purge somehow over the course of the Purge cycle?